src/Azure.Functions.Cli/Actions/DurableActions/DurableGetHistory.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using Fclp;
+using Azure.Functions.Cli.Interfaces;
+using Azure.Functions.Cli.Common;
+
+
+namespace Azure.Functions.Cli.Actions.DurableActions
+{
+    [Action(Name = "history", Context = Context.Durable, HelpText = "Gets the history of the instance of a specified orchestration instance")]
+    class DurableGetHistory : BaseAction
+    {
+        public string Version { get; set; }
+
+        public string Instance { get; set; }
+
+
+        private readonly ISecretsManager _secretsManager;
+        private readonly DurableManager durableManager;
+
+        public DurableGetHistory(ISecretsManager secretsManager)
+        {
+            _secretsManager = secretsManager;
+            durableManager = new DurableManager(secretsManager);
+        }
+
+
+        public override ICommandLineParserResult ParseArgs(string[] args)
+        {
+            var parser = new FluentCommandLineParser();
```

Comment:
since you're extending `BaseAction` you can use the `Parser` object that the base class is setting up for you instead of creating your own here.